addon/utils.js

```diff
@@ -7,6 +7,12 @@ export function isGeneratorIterator(iter) {
           typeof iter['throw']  === 'function');
 }
 
+export function isEventedObject(c) {
+  return (c &&
+          typeof c.one === 'function' &&
+          typeof c.off === 'function');
```

Comment:
@machty thats actually why I did it that way 😄 . If you want me to use the `subscribe` api, I can go ahead and switch to that. I just wasnt sure it would support the functionalities we are looking for.